packages/markdown-it-plugins/src/fence.ts

```diff
@@ -0,0 +1,59 @@
+import type MarkdownIt from "markdown-it";
+import type { Options } from "markdown-it";
+import type Token from "markdown-it/lib/token.mjs";
+import type Renderer from "markdown-it/lib/renderer.mjs";
```

Comment:
Seems weird that these can't all be one include, but that's just how this library has their stuff structured?

Reply:
I agree...  But at least they are only types (we don't bundle marked-it in our dist folder)